pkg/secrets/pushtofile/standard_templates_test.go

```diff
@@ -221,6 +232,43 @@ func testValidateAliasLenForJSON(t *testing.T) {
 	}
 }
 
+func testValidateAliasForProperties(t *testing.T) {
```

Comment:
In order for these tests to run, this function needs to be upcased:

```suggestion
func TestValidateAliasForProperties(t *testing.T) {
```